index.rst

```diff
@@ -556,6 +556,67 @@ may not serve in place of monkey patching the file: this caching feature may
 need to be optional (if the temporary file storage is getting too big), and it
 is possible that objects could still exist after the SwiftStorage is destroyed.
 
+Formatter
+---------
+
+A Formatter is an object that has the ability to serialize and deserialize
+objects to and from a type of persistent storage. ``StorageInterface``
+subclasses should use formatters to serialize & deserialize objects.
+
+A Formatter is registered for a class type with the ``StorageInterface``
```

Comment:
`for a class type` is a bit confusing, I had to think >3 seconds to guess that it means data object type (maybe because I did not read all previous sections). Can you try to make it a bit clearer?